Use git describe --tags --dirty as the versionName

grzesiek2010 opened the pull request:
This PR is in reference to resolve #305

grzesiek2010 committed:
Use git describe --tags --dirty as the versionName

grzesiek2010 committed:
Remove build number from main menu

lognaturel commented:
This works perfectly for me but of course I just realized that I forgot our Windows friends!! 😱

Adding Windows support should be as easy as adding a test that looks something like [this](https://gist.github.com/xconnecting/4158381). If you have a Windows VM or machine easily available to verify it on, that'd be great but if not just let me know and I'll do a check. 

No rush on this, I know you're going on vacation and it can wait until you get back. Thanks!

grzesiek2010 commented:
I added this but you are right I'm going on vacation and I'm not able to test this change on Windows now. I'm quite sure it's ok but please test ths if you want to merge that. Thanks.

And I implemented chnges for #238 too but that code is related to this a bit so I would like to avoid conflicts and I'll add that when you accept this. maybe today before my vacation :)

lognaturel commented:
Thanks, @ianume for the Windows check. :)
